sale_order_product_recommendation/tests/test_recommendation.py

```diff
@@ -87,12 +87,12 @@ def test_transfer(self):
             .with_context(active_ids=[self.new_so.id])
             .create(
                 {
-                    "advance_payment_method": "all",
+                    "advance_payment_method": "delivered",
```

Comment:
Why do you need to change this?

Reply:
https://github.com/odoo/odoo/blob/f2c214e227db7c9ec2340ef87644f2f7c2e370c5/addons/sale/wizard/sale_make_invoice_advance.py#L47

Reply:
OK, seen